### ISSUE TITLE
Very out of date 💀 

### ISSUE THREAD

**Issue by BlaNKtext** [web-osu/web-osu.github.io#2]
There has been quite a bit of updates made in the past 6 months since you updated it, we (the official community) have been getting some people that are confused which is the official site, I'm requesting you make some changes so people will know this is a mirror and not the real site. I am fine with this project existing as long as it is updated fairly regularly.

**Comment by RACECAR-00** [comment]
alr i made a github account js for saying this but im this dudes friend and he said he doesnt wanna manage it anymore. he says if he can find the password for it he will delete the entire account. the mirror was entirely made for school and nothing else and he didnt think anyone would find it so he was whatever with it.

anyways, this time with permission, can i make a mirror? jw since i can actually manage it

**Comment by RACECAR-00** [comment]
yes i know this is late but please make sure ppl are going to webosu.online for now and not this unmanaged one.

**Comment by BlaNKtext** [comment]
Yeah you are free to make a mirror, I use this one as a backup when our new api has downtime lol

**Comment by xzcollapse** [comment]
Anyone have the new mirror?
